### PR TITLE
Update to work with new config strategy

### PR DESCRIPTION
**Pull request by benbruning**
Idea would be to version this and move repos over to use it like `rhinoapi` and `rhinopay` (`rhinoorganization` is already using it) when moving to the new config api in the latest `rhinotilities`.

Due to the async nature of `RDSUtilities` these changes were needed.  

** Note: existing migrations will need to be flattened or updated for rebuilds to support the async db config.

### CODE REVIEW

File: lib/rhinozug.js [hunk 1]
```diff
@@ -352,8 +352,8 @@ function init() {
   }
 }
 
-function getConnection(config) {
-  return adapter.connect(config);
+async function getConnection(config) {
+  return adapter.connect((await config));
```
Comment (alexgsummer) [inline lib/rhinozug.js:356]: Looks like you're awaiting a variable here?

File: src/initFiles/seed.template [hunk 2]
```diff
@@ -4,12 +4,15 @@
 for documentation of the schema building api in use */
 
 let rz = require('rhinozug');
-let config = require(`../config/${process.env.connection}.js`);
+
+async function getConfig() {
```
Comment (alexgsummer) [inline src/initFiles/seed.template:8]: `async` not needed here, since there are no `await`s

File: src/rhinozug.js [hunk 3]
```diff
@@ -352,8 +352,8 @@ export function init() {
   }
 }
 
-export function getConnection(config) {
-  return adapter.connect(config);
+export async function getConnection(config) {
+  return adapter.connect(await config);
```
Comment (alexgsummer) [inline src/rhinozug.js:356]: Same here, this is `await`ing a variable rather than a function call.

Comment (benbruning) [reply src/rhinozug.js:356]: @alexgsummer `config` here is an unresolved Promise due to the new RDSUtilities.

**Pull request merged**